app/src/main/java/com/garan/wearwind/screens/Connected.kt

```diff
@@ -84,7 +95,12 @@ fun ConnectedScreen(
 }
 
 @Composable
-fun SpeedAndHrBox(uiState: UiState, screenStarted: Boolean, hr: Int, speed: Int) {
+fun SpeedAndHrBox(
+    uiState: UiState,
+    screenStarted: Boolean,
+    metricDisplayTypography: MetricTypography = MetricTypography(),
```

Comment:
These are clear defaults, and reused.  But hard to actually adjust without passing it down through all layers.  HAve you considered a Composition Local? https://developer.android.com/jetpack/compose/compositionlocal